### PR TITLE
Package release

### PR DESCRIPTION
**Pull request by serban-nicusor-toptal**
#### Submission Checklist

- [ ] Run unit tests
- [ x ] Declare copyright holder and open-source license: see below

#### Summary
PR that handles issue #295 of release and documentation.
We're automating this process through a Jenkins job, note that readthedocs call to change the default version is done through python requests and session rather than an API request ( they do not have one for the default version ).

The process is:
- Create a new branch for the release, release/v1.0.0 for example
- Update version in cmdstanpy/_version.py
- Generate docs in docsrc with make github
- Push the new branch to upstream
- Merge new branch into develop
- Create a new tag with the release version
- Update master branch to the new version
- Install, build and push to pypi
- Change default version in readthedocs.

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Toptal

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)

### CODE REVIEW

File: rtd_change_default_version.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            'conf_py_file': 'docsrc/conf.py',
+            'enable_pdf_build': 'on',
+            'enable_epub_build': 'on',
+            'save': 'Save'
+        }
+
+        # Post our version data
+        r = s.post(url, data = version_data, headers=dict(Referer=url))
+
+set_version_active_on_rtd(sys.argv[1], sys.argv[2], sys.argv[3], sys.argv[4])
```
Comment (ahartikainen) [inline rtd_change_default_version.py:56]: Can this be added under __name__ main block:

    ```suggestion
if __name__ == '__main__':
    set_version_active_on_rtd(sys.argv[1], sys.argv[2], sys.argv[3], sys.argv[4])
```

Comment (serban-nicusor-toptal) [reply rtd_change_default_version.py]: Done

**Comment by serban-nicusor-toptal** [comment]
I've refactored a bit and tested, this is ready for review.

**Comment by mitzimorris** [comment]
I defer to @ahartikainen on this one

**Review by ahartikainen — APPROVED** [review]
LGTM

**Comment by mitzimorris** [comment]
many thanks to both of you - this is awesome!

**Pull request merged**